### ISSUE TITLE
Slurm doesn't use pmix in ohpc v2.1

### ISSUE THREAD

**Issue by sjpb** [openhpc/ohpc#1320]
In OpenHPC v2.0, you can do:

```
srun --mpi=pmix_v3
```

and it just works. Turns out `pmix-ohpc` is listed as a dependency for slurm:

```
rpm -q --whatrequires pmix-ohpc
slurm-ohpc-20.02.5-13.1.ohpc.2.0.x86_64
```

In OpenHPC v2.1 this doesn't work, and even installing the `pmix-ohpc` package doesn't fix that.

@adrianreber suggested it's related to this which changes the pmix conditional: https://github.com/openhpc/ohpc/commit/c04c33f48d2a75e7fe07697ce277a981926c61df 

#1273 may also be related if pmix support is supposed to be done a different way?

**Comment by tuxwielder** [comment]
Looks like this is caused by rpath-checks in the Spec-file (on RHEL-based OS). If I use:
`export QA_RPATHS=$(( 0x0001|0x0002|0x0004|0x0005 ))`
in stead of:
`export QA_RPATHS=0x5`
then the pmix module seems to get installed..

**Comment by yixuan178** [comment]
any progress for that issue reported for 2 months ago? will you update spec to resolve it?

**Comment by koomie** [comment]
We will endeavor to re-enable PMIx support in the 2.5 release. The pmix packaging needs to be udpated.

**Comment by adrianreber** [comment]
Should be fixed with 2.8
